src/SidebarProvider.ts

```diff
@@ -46,8 +47,21 @@ export class SidebarProvider implements vscode.WebviewViewProvider {
             vscode.commands.executeCommand(
               "workbench.files.action.showActiveFileInExplorer"
             );
+            // Run code to automatically install files in a folder
+            if (process.env.INSTALL_COMMAND) {
+              this._runCommand(process.env.INSTALL_COMMAND);
+            }
           }
+          break;
+        }
 
+        case "endInterview": {
+          vscode.window.showInformationMessage(
+            "Successfully ended your interview. You can leave this session."
+          );
+          this._runCommand(
+            `git add . && git commit -m 'Upload solution' && git push`
+          );
```

Comment:
TODO: update the LiveCandidateAssessment's status here? 

Unless we have some kind of webhook listening in to when a commit is pushed on here that is related to the end interview?